spec/models/service_spec.rb

```diff
@@ -13,4 +13,62 @@
   it { is_expected.to allow_value(nil).for(:tier) }
 
   it { is_expected.to have_many(:service_interactions) }
+
+  describe '#provided_by?' do
+    let(:district) { FactoryGirl.build(:local_authority, tier: 'district') }
+    let(:county) { FactoryGirl.build(:local_authority, tier: 'county') }
+    let(:unitary) { FactoryGirl.build(:local_authority, tier: 'unitary') }
+
+    context 'when the tier is blank' do
+      subject { FactoryGirl.build(:service, tier: nil) }
+      it 'is false for a district council' do
+        expect(subject).not_to be_provided_by(district)
```

Comment:
Yeah, for any `predicate?` method on an instance rspec lets you use `be_predicate` as the matcher: https://www.relishapp.com/rspec/rspec-expectations/v/3-4/docs/built-in-matchers/predicate-matchers